test/tools/Dockerfile

```diff
@@ -46,3 +46,8 @@ RUN aws --version
 # Install eksctl
 RUN curl --silent --location "https://github.com/weaveworks/eksctl/releases/download/v0.105.0/eksctl_$(uname -s)_amd64.tar.gz" | tar xz -C /tmp
 RUN mv /tmp/eksctl /usr/local/bin
+
+# Install toolchain
```

Comment:
seems like you're just caching the modules not installing the "toolchain"?

Reply:
Good point. I was originally going to install the toolchain, but it took foreverrrrr